src/app-admin.js

```diff
@@ -0,0 +1,444 @@
+import express from 'express';
+import * as uuid from 'uuid';
+import {
+  errorMiddleware,
+  requestLoggerMiddleware,
+  validateSessionMiddleware,
+} from './util/middlewares.js';
+import validateSession from './util/validate-user.js';
+import { isAdmin } from './account-db.js';
+import config from './load-config.js';
+import * as UserService from './services/user-service.js';
+
+let app = express();
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(errorMiddleware);
+app.use(requestLoggerMiddleware);
+
+export { app as handlers };
+
+app.get('/ownerCreated/', (req, res) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
app.get('/owner-created/', (req, res) => {
```

Reply:
changed